public/app/plugins/datasource/elasticsearch/elastic_response.ts

```diff
@@ -410,4 +412,140 @@ export class ElasticResponse {
 
     return { data: seriesList };
   }
+
+  getLogs(logMessageField?: string, logLevelField?: string): DataQueryResponse {
```

Comment:
Would be nice with a test verifying that a response of both documents and date histogram aggregation can be properly parsed.